### PR TITLE
fix: Pulled dev branch in next-release.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
fix: Pulled dev branch in next-release.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-rest-api.php [hunk 1]
```diff
@@ -53,6 +57,7 @@ public function __construct() {
 		 * @since 1.23.0
 		 */
 		public function delete_page_assets( $post_id ) {
+			error_log('in delete_page_assets ');
```
Comment (sandeshjangam) [inline classes/class-uagb-rest-api.php:57]: DO we need this?

File: classes/class-uagb-rest-api.php [hunk 2]
```diff
@@ -45,6 +45,10 @@ public function __construct() {
 
 			// We have added this action here to support both the ways of post updations, Rest API & Normal.
 			add_action( 'save_post', array( $this, 'delete_page_assets' ), 10, 1 );
+
+			add_action( 'ast_after_widget_save', function() {
+				error_log('in ast_after_widget_save ');
+			});
```
Comment (sandeshjangam) [inline classes/class-uagb-rest-api.php]: Remove it

File: classes/class-uagb-rest-api.php (hunk 1 above)
Comment (RajkiranBagal) [reply classes/class-uagb-rest-api.php]: removed

File: classes/class-uagb-rest-api.php (hunk 2 above)
Comment (RajkiranBagal) [reply classes/class-uagb-rest-api.php]: removed

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**